meson.build

```diff
@@ -432,11 +432,11 @@ ladspa_symbol_args = []
 lv2_symbol_args = []
 vamp_symbol_args = []
 
-if get_option('buildtype').startswith('release')
+if get_option('debug')
+  config_summary += { 'Build type': 'Debug' }
```

Comment:
To avoid confusion with meson's `buildtype`, do you think it might be better to output "Debug: enabled" or "Debugging: enabled" instead?